tiledb/bioimg/helpers.py

```diff
@@ -456,3 +458,31 @@ def merge_ned_ranges(
     merged_ranges_per_axis = [merge_ranges(ranges) for ranges in ranges_per_axis]
 
     return tuple(merged_ranges_per_axis)
+
+
+def remove_ome_image_metadata(xml_string: str) -> Union[str, Any]:
```

Comment:
doc strings if this needs to be rendered in an api ref eventually

Reply:
Added docstrings for the `remove_ome_image_metadata ` function although it is an internal function. Nice habits.!